bats/tests/helpers/os.bash

```diff
@@ -2,6 +2,7 @@
 # shellcheck disable=SC2120
 UNAME=$(uname)
 ARCH=$(uname -m)
+ARCH=${ARCH/arm64/aarch64}
```

Comment:
I was thinking about using `case` or `if` to make it clearer, but I don't see any arch that has `arm64` as a substring, so this is probably okay for now.